configs/BentoBox/BoardConfig.h

```diff
@@ -38,7 +38,7 @@
 #define PIN_BUTTON_TURBO -1         // Turbo
 #define PIN_BUTTON_REVERSE -1       // UDLR Reverse
 #define PIN_SLIDER_LS    -1         // Left Stick Slider
```

Comment:
You missed this instance of `PIN_SLIDER_LS` during renaming.

Reply:
Oh, this is strange, thank you for catching it! Fixed!